PowerShellTools/DebugEngine/PromptUI/ReadHostPromptDialogViewModel.cs

```diff
@@ -0,0 +1,89 @@
+﻿using PowerShellTools.Common.Debugging;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.DebugEngine.PromptUI
+{
+    public class ReadHostPromptDialogViewModel : INotifyPropertyChanged
+    {
+        private string _parameterValue;
+        private string _parameterMessage;
+        private string _parameterName;
+        private string _title;

```

Comment:
Modified, actually it could be parameter name in parameter prompt, and for read-host, it is just read-host const title string.